### PR TITLE
Display voice broadcast total length

### PR DESCRIPTION
**Pull request by weeman1337**
Part of https://github.com/vector-im/element-web/issues/23282

## Checklist

* [x] Tests written for new code (and old code if feasible)
* [x] Linter and other CI checks pass
* [x] Sign-off given on the changes (see [CONTRIBUTING.md](https://github.com/matrix-org/matrix-react-sdk/blob/develop/CONTRIBUTING.md))

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

### CODE REVIEW

**Comment by weeman1337** [comment]
Back to draft an clearing review, because there will be some refactoring

File: src/voice-broadcast/models/VoiceBroadcastPlayback.ts [hunk 1]
```diff
@@ -285,15 +297,13 @@ export class VoiceBroadcastPlayback
...
-    }
-
     public destroy(): void {
         this.chunkRelationHelper.destroy();
         this.infoRelationHelper.destroy();
         this.removeAllListeners();
-        this.destroyQueue();
+
+        this.chunkEvents = new VoiceBroadcastChunkEvents();
+        this.playbacks.forEach(p => p.destroy);
```
Comment (florianduros) [inline src/voice-broadcast/models/VoiceBroadcastPlayback.ts:306]: Is it intentional to not call `p.destroy` here ?

**Review by florianduros — CHANGES_REQUESTED** [review]
One question about the `destroy` :)

**Review by florianduros — APPROVED** [review]
Looks good!

**Pull request merged**